lib/common/exceptions/external/backend_service_exception.rb

```diff
@@ -1,25 +1,52 @@
 # frozen_string_literal: true
 module Common
   module Exceptions
-    # BackendServiceException - This will eventually receive a Common::Client::Errors:BackendServiceError
-    # It should not be invoked directly by clients at any time. The Client:Error should be
-    # rescued and this should be raised instead.
+    # BackendServiceException - This will return a generic error, to customize
+    # you must define the minor code in the locales file and call this class from
+    # raise_error middleware.
     class BackendServiceException < BaseError
-      def initialize(error_klass = nil, options = {})
-        @error_klass = error_klass || 'client_error'
-        @detail = options[:detail]
-        @source = options[:source]
-        @meta = options[:meta] unless Rails.env.production?
+      attr_reader :response_values
+
+      def initialize(key = nil, response_values = {})
+        if response_values[:status]&.between?(400, 999)
+          @response_values = response_values
+        else
+          raise NotImplementedError, 'only invoke from raise_error middleware'
```

Comment:
Yea, this is kind of a dumb heuristic. It would be a violation of OOP to reference the caller, don't want to do that. Maybe I can make the error message be something else. The idea here is though, that one shouldn't mess with raising this exception outside of the middleware, but I can't think of a good way of enforcing that. Maybe just being deligent in code review. Not sure it's even necessary as long as its done correctly -- maybe i'll remove this check entirely.